### PR TITLE
[ONNX] Bump onnx-script version with imported module renaming

### PR DESCRIPTION
**Pull request by titaiwangms**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #99927
* __->__ #99926

Avoid potential blocking after https://github.com/microsoft/onnxscript/pull/659

### CODE REVIEW

File: .ci/docker/common/install_onnx.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ pip_install \
   transformers==4.25.1
 
 # TODO: change this when onnx-script is on testPypi
-pip_install "onnx-script@git+https://github.com/microsoft/onnxscript@1e8d764a9be04323d7171e4d5f511332790cb809"
+pip_install "onnx-script@git+https://github.com/microsoft/onnxscript@99a665546d374af872cf136137b5d8334389b78d"
```
Comment (justinchuby) [inline .ci/docker/common/install_onnx.sh:30]:
```suggestion
pip_install "onnxscript@git+https://github.com/microsoft/onnxscript@99a665546d374af872cf136137b5d8334389b78d"
```

it was renamed

**Review by justinchuby — APPROVED** [review]

**Comment by justinchuby** [comment]
I Will wait for you to merge this before #99284

**Review by BowenBao — APPROVED** [review]

**Comment by titaiwangms** [comment]
> I will wait for you to merge this before #99284

Sorry, I will try merge it tonight!

**Pull request closed without merging**